Cloud release notes april 2024 (#6043)
umbraco/UmbracoDocs: umbraco-cloud/release-notes/2024-04-releasenotes.md

# April 2024

## Key Takeaways

* **New Project UI** - New intuitive and modern user interface for projects on Umbraco Cloud.

## New Project UI

> From creating new environments, managing your team, or gaining insight into the performance of your project it has never been easier to use Umbraco Cloud.

Reviewer: ```suggestion
From creating new environments, managing your team, to gaining insight into the performance of your project it has never been easier to use Umbraco Cloud.
```

> With the new project UI and UX on Umbraco Cloud, we have made it simpler and more intuitive to manage your project on Umbraco Cloud.

Reviewer: ```suggestion
With the new UI and UX for projects on Umbraco Cloud, managing your project is now more straightforward and intuitive.
```

When visiting your project, you'll notice that your project now has a sidebar, containing the different project settings.

> The environment overview have also gotten a new design as well, making it easier to manage your different environments.

Reviewer: ```suggestion
The environment overview has received a fresh design, making it easier to manage your different environments.
```

> With this design, we have created a new and  modern UI and UX making it even easier to manage your project.

Reviewer: ```suggestion
```

This line is already added above on line 12.
